app/assets/javascripts/admin/addon/templates/embedding-index.hbs

```diff
@@ -0,0 +1,45 @@
+{{#if this.embedding.embeddable_hosts}}
+  <table class="d-admin-table">
+    <thead>
+      <th>{{i18n "admin.embedding.host"}}</th>
+      <th>{{i18n "admin.embedding.allowed_paths"}}</th>
+      <th>{{i18n "admin.embedding.category"}}</th>
+      <th>{{i18n "admin.embedding.tags"}}</th>
+      {{#if this.embedding.embed_by_username}}
+        <th>{{i18n
+            "admin.embedding.post_author"
+            author=this.embedding.embed_by_username
+          }}</th>
+      {{else}}
+        <th>{{i18n "admin.embedding.post_author"}}</th>
+      {{/if}}
+    </thead>
+    <tbody>
+      {{#each this.embedding.embeddable_hosts as |host|}}
+        <EmbeddableHost @host={{host}} @deleteHost={{action "deleteHost"}} />
+      {{/each}}
+    </tbody>
+  </table>
+{{else}}
+  <AdminConfigAreaEmptyList
+    @ctaLabel="admin.embedding.add_host"
+    @ctaRoute="adminEmbedding.new"
+    @ctaClass="admin-embedding__add-host"
+    @emptyLabel="admin.embedding.get_started"
+  />
+{{/if}}
+
+<PluginOutlet
+  @name="after-embeddable-hosts-table"
+  @outletArgs={{hash embedding=this.embedding}}
+/>
+
+{{#if this.showEmbeddingCode}}
```

Comment:
Maybe instead of in details/summary, you could use `AdminConfigAreaCard` with your new expand/collapse feature, and have it collapsed by default?

Might also be worth having this above the table, otherwise it's hard to find when more records show